fixieai/cli/agent/commands.py

```diff
@@ -538,11 +532,22 @@ def _add_text_file_to_tarfile(path: str, text: str, tarball: tarfile.TarFile):
     help="(default enabled) Validate that the agent loads in a local venv before deploying",
 )
 @click.pass_context
-def deploy(ctx, path, metadata_only, validate):
+def deploy(ctx, path, metadata_only, public, validate):
     console = rich_console.Console(soft_wrap=True)
     config = agent_config.load_config(path)
+    if config.public is not None:
+        console.print(
+            "[yellow]Warning:[/] The [blue]`public`[/] option in the agent config is deprecated and will be removed soon."
+        )
+        console.print(
+            "[yellow]Warning:[/] Use [blue]`fixie agent deploy --public`[/] instead."
```

Comment:
Maybe I'm confused, but is the correct syntax `fixie agent deploy --public` or `fixie agent publish`? Or both?

Reply:
Both. You can use `fixie agent deploy --public` to deploy+publish in one step, or use `fixie agent publish` on an already-deployed agent.